src/ui/views/ui_app.jade

```diff
@@ -125,7 +125,7 @@ block content
           err.style['text-align'] = 'center';
           err.style['background'] = '#F44336';
           err.style['padding'] = '25px';
-          err.innerText = '#{i18n('UI-WELCOME_ERROR')}';
+          err.innerText = '#{i18n_singleQuoteVar('UI-WELCOME_ERROR')}';
 
```

Comment:
@hickeyma I tried that, actually using `JSON.stringify` instead and double quotes… but Jade (now pug) ends up quoting oddly- I think a html safe transform is implicated.